opm/models/utils/simulator.hh

```diff
@@ -99,6 +99,8 @@ class Simulator
     using MPIComm = typename Dune::MPIHelper::MPICommunicator;
     using Communication = Dune::Communication<MPIComm>;
 
+    static constexpr double eps = std::max(std::numeric_limits<Scalar>::epsilon(), 1.e-9);
```

Comment:
some value between 1.e-9 and 1.e-19 can get correct behavoir for `[GASCONDENSATE_VAPWAT_PRECSALT_REGRESSION](https://ci.opm-project.org/job/opm-simulators-PR-builder/7846/testReport/junit/(root)/mpi/compareECLFiles_flow_GASCONDENSATE_VAPWAT_PRECSALT_REGRESSION/) `